Makefile

```diff
@@ -11,10 +13,19 @@ all: get_tools get_vendor_deps install install_examples test_lint test
 ci: get_tools get_vendor_deps install test_cover test_lint test
 
 ########################################
-### Build
+### Build/Install
+
+check-ledger: 
+ifeq ($(LEDGER_ENABLED),true)
+ifndef GCC
```

Comment:
Should we add a check here for the linux udev rules? 
https://support.ledgerwallet.com/hc/en-us/articles/115005165269-Connection-issues-with-Windows-or-Linux

Reply:
Mhmmm, not sure? But feel free to update it if it makes sense :-)